docs/app/Examples/modules/Progress/Content/ProgressExampleLabel.js

```diff
@@ -1,8 +0,0 @@
-import React from 'react'
-import { Progress } from 'semantic-ui-react'
-
-const ProgressExampleLabel = () => (
-  <Progress percent={55}>Label</Progress>
```

Comment:
Let's return this example, it shows another behaviour. However, there is #1355 that fixes bug, so we need a small update there:

```diff
-<Progress percent={55}>Label</Progress>
+<Progress percent={55} label>Label</Progress>
```